src/js/actions/type.js

```diff
@@ -817,7 +817,7 @@ define(function (require, exports) {
     beforeStartup.action = {
         reads: [],
         writes: [],
-        modal: []
+        modal: true
```

Comment:
Potential future work: manually check the types of these properties to guard against typos like the ones in this file.